### PR TITLE
feat: add a pr assign config var

### PR DESCRIPTION
**Pull request by omercnet**
fixes https://github.com/pulumi/upgrade-provider/issues/121

### CODE REVIEW

**Comment by rquitales** [comment]
The changes here look good to me, but I'm unfamiliar with the codebase, so will defer to other team members for final approval.

File: main.go [hunk 1]
```diff
@@ -209,6 +209,9 @@ Required unless running from provider root and set in upgrade-config.yml.`)
 	cmd.PersistentFlags().StringVar(&context.PrReviewers, "pr-reviewers", "",
 		`A comma separated list of reviewers to assign the upgrade PR to.`)
 
+	cmd.PersistentFlags().StringVar(&context.PrAssign, "pr-assign", "@me",
+		`A user to assign the upgrade PR to (default: @me).`)
```
Comment (iwahbe) [inline main.go:213]: The default is already included by cobra, so this renders as 

```
--pr-assign string        A user to assign the upgrade PR to (default: @me). (default "@me")
```

```suggestion
	cmd.PersistentFlags().StringVar(&context.PrAssign, "pr-assign", "@me",
		`A user to assign the upgrade PR to.`)
```

**Review by iwahbe — CHANGES_REQUESTED** [review]
Hi @omercnet. Thanks for the PR!

To finish up, you will need to thread `Context.PrAssign` to where `"@me"` is currently used:

https://github.com/pulumi/upgrade-provider/blob/cb42bff41add5feada028094f5fc02922fbdbd0b/upgrade/steps.go#L313-L315

Right now, your changes give the user the option to set their desired assignee but doesn't actually respect that option.

**Review by iwahbe — APPROVED** [review]
This looks good to me.

**Pull request merged**